charts/tfy-karpenter-config/README.md

```diff
@@ -26,15 +27,23 @@ version_lte() {
 version_lt() {
   ! version_lte "$2" "$1"
 }
+
 CONTAINERD_VERSION_OUTPUT="$(containerd --version)"
 IFS=" " read -a  CONTAINERD_VERSION_STRING <<< "${CONTAINERD_VERSION_OUTPUT}"
 CONTAINERD_VERSION=${CONTAINERD_VERSION_STRING[2]}
-echo containerd version "${CONTAINERD_VERSION}"
+SANDBOX_IMAGE=$(containerd config dump | grep -oE 'sandbox_image = "(.*)"' | sed 's/sandbox_image = //')
+KUBELET_CONFIG_DIR=/etc/kubernetes/kubelet/config.json.d
+SOCI_KUBELET_CONFIG_PATH=$KUBELET_CONFIG_DIR/99-soci.conf
```

Comment:
Is this always 99-soci.conf?

Reply:
You can have multiple configurations that will be merged.

```
[root@ip-10-2-17-231 /]# ls /etc/kubernetes/kubelet/config.json.d
00-nodeadm.conf  99-soci.conf
```

nodeadm adds the default and we patch on top of that.

Reply:
The name does not need to be `99-soci.conf`.